### ISSUE TITLE
No support for overloading extern functions (not methods)

### ISSUE THREAD

**Issue by hackedy** [cornell-netlab/petr4#52]
An up-to-date v1model.p4 includes the following overloaded extern function.
```
extern void log_msg(string msg);
extern void log_msg<T>(string msg, in T data);
```
I know the typechecker won't handle this appropriately, but the parser also appears to be having trouble with it. I get this cryptic error message whenever I try to typecheck programs that use v1model.
```
[exception] Petr4.Parser.MenhirBasics.Error
Raised at file "map.ml", line 135, characters 10-25
Called from file "sexp_conv.ml", line 147, characters 10-37
```

**Comment by hackedy** [comment]
Sorry, I think this is actually an issue with parsing the type `string`.

**Comment by hackedy** [comment]
Aaand It's been fixed  since d95e18f5 and I just need to merge master into `typed-ast`. Never mind!
